api/types/container/host_config.go

```diff
@@ -284,6 +284,17 @@ type DeviceMapping struct {
 	CgroupPermissions string
 }
 
+// HugepageLimit corresponds to the file`hugetlb.<hugepagesize>.limit_in_byte` in container level cgroup.
+// For example, `PageSize=1GB`, `Limit=1073741824` means setting `1073741824` bytes to hugetlb.1GB.limit_in_bytes.
+type HugepageLimit struct {
+	// The value of PageSize has the format <size><unit-prefix>B (2MB, 1GB),
+	// and must match the <hugepagesize> of the corresponding control file found in `hugetlb.<hugepagesize>.limit_in_bytes`.
+	// The values of <unit-prefix> are intended to be parsed using base 1024("1KB" = 1024, "1MB" = 1048576, etc).
+	PageSize string
+	// limit in bytes of hugepagesize HugeTLB usage.
+	Limit uint64
```

Comment:
Can you use int64 instead?

Reply:
`Limit` follows `LinuxResources`, it has uint64 for hugepage limit.
Please tell me if there is another reason that i didn't catch.

Reply:
It's more in line with (most) of the rest of the API.